html_reference.html

```diff
@@ -4,6 +4,7 @@
 		<link rel="stylesheet" href="css_stylesheet.css">
```

Comment:
Consider adding the lang="en-GB" attribute to your html element

Reply:
added